Automattic/wp-calypso
# Hosting Metrics: Add minor improvements to the first protype

Closes https://github.com/Automattic/dotcom-forge/issues/3259

## Proposed Changes

This PR addresses the comments from https://github.com/Automattic/dotcom-forge/issues/3259 and incorporates the suggested feedback. More specifically:

* it specifies the types for `const formattedData`

## Testing Instructions

* Navigate to https://github.com/Automattic/wp-calypso/pull/79471 and review the feedback there
* Check the code changes in this PR and confirm the suggestions were implemented
* Confirm that the graph still works as expected by following these steps:

1. Make sure you have an Atomic site
2. Navigate to Calypso live link and amend `site-monitoring/{yourAtomicsite}`
3. View the graph that represents HTTP requests per second on the site in the last 24 hours

## Pre-merge Checklist
